src/main/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepository.kt

```diff
@@ -16,10 +17,17 @@ interface CertificateRepository : JpaRepository<Certificate, UUID> {
 
     fun findByStatusAndPrintRequestsBatchId(certificateStatus: Status, batchId: String): List<Certificate>
 
-    fun findByStatus(certificateStatus: Status): List<Certificate>
+    fun findByStatusOrderByApplicationReceivedDateTimeAsc(certificateStatus: Status, pageable: Pageable): List<Certificate>
 
     fun findByGssCodeInAndSourceTypeAndSourceReference(gssCodes: List<String>, sourceType: SourceType, sourceReference: String): Certificate?
 
-    @Query("SELECT COUNT(*) FROM PrintRequestStatus s WHERE s.eventDateTime >= :startInstant AND s.eventDateTime <= :endInstant AND s.status = :status")
```

Comment:
My IDE was complaining that `COUNT(*)` was not value JPQL (even though it was working fine). I changed it to `COUNT(s)` to keep my IDE quiet! 😁